src/index.css

```diff
@@ -1,5 +1,5 @@
 body {
```

Comment:
- [ ] OPTIONAL
Your **Pull Request** is indicating that `src/index.css` is the only file you worked on. Kindly endeavor to make the necessary _git commits_ for this project.